autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/selecting/AutofillSelectCredentialsDialogFragment.kt

```diff
@@ -28,6 +28,8 @@ import androidx.lifecycle.lifecycleScope
 import com.duckduckgo.anvil.annotations.InjectWith
 import com.duckduckgo.app.browser.favicon.FaviconManager
 import com.duckduckgo.app.global.extractDomain
+import com.duckduckgo.app.statistics.pixels.Pixel
+import com.duckduckgo.app.pixels.AppPixelName
```

Comment:
This import is not working. Not sure how I should do this.

Reply:
Replace `import com.duckduckgo.app.pixels.AppPixelName` with your own enum where the pixel is defined (see previous comment)